addons/l10n_es/data/product_data.xml

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
```

Comment:
These are predefined products, such that when people need to enter a customs invoice as a vendor bill, they just put that product on the line for the valuation of the product by the customs.  It makes sense to update in case they would delete or change by accident.

Reply:
Delete with noupdate recreates, but ok for the changes